hack/lib/common.sh

```diff
@@ -706,6 +707,22 @@ function kube::common::start_kubelet {
       image_service_endpoint_args=("--image-service-endpoint=${IMAGE_SERVICE_ENDPOINT}")
     fi
 
+    KUBELET_FLAGS="--tenant-server-kubeconfig="
+    if [[ "${IS_SCALE_OUT}" == "true" ]] && [ "${IS_RESOURCE_PARTITION}" == "true" ]; then
+      serverCount=${#TENANT_SERVERS[@]}
+      kubeconfig_filename="tenant-server-kubelet"
+      for (( pos=0; pos<${serverCount}; pos++ ));
+      do
+        # here generate kubeconfig for remote API server. Only work in non secure mode for now
+        kube::util::write_client_kubeconfig "${CONTROLPLANE_SUDO}" "${CERT_DIR}" "" "${TENANT_SERVERS[${pos}]}" "${API_PORT}" tenant-server-kubelet "" "http"
```

Comment:
to clarify: how this will be updated if secure port is used , with cert and token needs to be set ?

Reply:
Not sure how certs can be used in local cluster up and also allowing either TP or RP starts arbitrarily. We might need to use token.

> to clarify: how this will be updated if secure port is used , with cert and token needs to be set ?